### DISCUSSION TITLE
Limit the quantity of candidates

### DISCUSSION THREAD

**Discussion by alienbogart** [company-mode/company-mode#1197] [Q&A] [answered]
#### Runtime Environment

- MX Linux 17
- GNU Emacs 27.0.50 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.22.11)
of 2018-11-28
- Evil version 1.2.13
- [.emacs.d](https://github.com/mrbig033/dotfiles/tree/master/emacs/emacs_default)
- [packages](https://github.com/mrbig033/dotfiles/blob/master/emacs/emacs_default/packs.el)
- company-20181207.147

Sometimes `company` gives too much possible completions (like ten or so), which I find unpractical. Is it possible to limit the quantity of candidates presented by it?

**Comment by dgutov** [comment, marked as answer]
`company-tooltip-limit`?

**Comment by alienbogart** [comment]
Thank you very much. I spend a lot of time trying to find this variable, tried all the obvious combination with the "candidate(s)" keyword!

**Comment by dgutov** [comment]
:ok_hand: 

Like the homepage mentions, there is an easy way to find all user options: `M-x customize-group RET company RET`.

**Comment by alienbogart** [comment]
Cool. I was searching via Ivy/Counsel, which is very handy, but the customize visualization would certainly be helpful. Thanks again.
